apps/api/Dockerfile

```diff
@@ -3,34 +3,33 @@ ENV PNPM_HOME="/pnpm"
 ENV PATH="$PNPM_HOME:$PATH"
 LABEL fly_launch_runtime="Node.js"
 RUN corepack enable
-COPY . /app
+
 WORKDIR /app
 
+# Install dependencies only
 FROM base AS prod-deps
+COPY package.json pnpm-lock.yaml ./
 RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install --prod --frozen-lockfile
 
+# Install all dependencies and build the application
 FROM base AS build
-RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install --frozen-lockfile
-
-RUN pnpm install
-RUN pnpm run build
-
-# Install packages needed for deployment
-
-
-FROM base
-RUN apt-get update -qq && \
-    apt-get install --no-install-recommends -y chromium chromium-sandbox && \
```

Comment:
@nickscamara do we need chromium-sandbox?

Reply:
I don't think so.

Reply:
Thanks, will remove that